service-worker/basic/service-worker.js

```diff
@@ -1,12 +1,76 @@
-self.addEventListener('fetch', function(event) {
-  console.log('got a request');
+/*
+ Copyright 2016 Google Inc. All Rights Reserved.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
 
-  var salutation = 'Hello, ';
-  var whom = decodeURIComponent(event.request.url.match(/\/([^/]*)$/)[1]);
-  var energyLevel = (whom === 'Cleveland') ? '!!!' : '!';
-  var version = '\n\n(Version 1)';
+const CACHES = {
```

Comment:
Feels weird to have a `CACHES` as it feels conflicty with `window.caches`. Makes it harder to search for each. 

Could we rename or just use two basic string variables instead?